tldw_Server_API/app/core/Metrics/metrics_manager.py

```
@@ -142,15 +143,28 @@ def _normalize_label_name(cls, name: str) -> str:
         if not labels:
             return {}
         normalized: dict[str, str] = {}
         for key, value in labels.items():
             normalized_key = cls._normalize_label_name(str(key))
             normalized_value = "" if value is None else str(value)
             if normalized_key in normalized and normalized[normalized_key] != normalized_value:
+                if reject_collisions:
+                    raise ValueError(
+                        "Conflicting labels normalize to the same key "
+                        f"{normalized_key!r}: {normalized[normalized_key]!r} vs {normalized_value!r}"
+                    )
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)